### PR TITLE
chore: update build process

### PR DESCRIPTION
**Pull request by aanorbel**

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -5,7 +5,7 @@ android-compileSdk = "34"
 android-minSdk = "24"
 android-targetSdk = "34"
 
-compose-plugin = "1.7.0-alpha02"
+compose-plugin = "1.6.11"
```
Comment (sdsantos) [inline gradle/libs.versions.toml:8]: We can go back to the stable version? Nice!

File: composeApp/build.gradle.kts [hunk 2]
```diff
@@ -173,9 +166,10 @@ tasks.register("cleanCopiedCommonResourcesToFlavor") {
     doLast {
         val projectDir = project.projectDir.absolutePath
 
-        val destinationFile = File(projectDir, config.resRoot)
+        val destinationFile = File(projectDir, "src/commonMain/composeResources")
         destinationFile.listFiles()?.forEach { folder ->
             folder.listFiles()?.forEach { file ->
+                println(file.absoluteFile)
```
Comment (sdsantos) [inline composeApp/build.gradle.kts:172]: Is this still needed?

File: composeApp/build.gradle.kts [hunk 3]
```diff
@@ -279,3 +273,15 @@ fun copyRecursive(
         }
     }
 }
+
+tasks.register("runDebug", Exec::class) {
```
Comment (sdsantos) [inline composeApp/build.gradle.kts:277]: Why do we need this task?

**Review by sdsantos — APPROVED** [review]

File: .github/workflows/validate.yml [hunk 4]
```diff
@@ -21,7 +21,7 @@ jobs:
         uses: ./.github/actions/setup
 
       - name: Build Android
-        run: ./gradlew assemble${{ matrix.type }}
+        run: ./gradlew clean assemble${{ matrix.type }}
```
[the remark this answers is not in the document]
Comment (aanorbel) [reply .github/workflows/validate.yml:24]: Yes we can run `copyCommonResourcesToFlavor`

File: composeApp/build.gradle.kts (hunk 2 above)
Comment (aanorbel) [reply composeApp/build.gradle.kts:172]: Debug log. will remove

Comment (aanorbel) [reply composeApp/build.gradle.kts:172]:
```suggestion
```

File: composeApp/build.gradle.kts (hunk 3 above)
Comment (aanorbel) [reply composeApp/build.gradle.kts:276]: This enables us to set up custom gradle tasks with `composeApp:runDebug -Porganization=ooni` or `composeApp:runDebug -Porganization=dw` to run the various flavors without needing to edit files.

Comment (aanorbel) [reply composeApp/build.gradle.kts:276]: I haven't tried any option that works

File: composeApp/build.gradle.kts [hunk 5]
```diff
@@ -279,3 +272,15 @@ fun copyRecursive(
...
+    dependsOn("clean", "uninstallDebug", "installDebug")
```
[the remark this answers is not in the document]
Comment (aanorbel) [reply composeApp/build.gradle.kts:277]: The pre build works locally but for some reason it fails on the ci

[the remark this answers is not in the document]
Comment (sdsantos) [reply composeApp/build.gradle.kts:277]: If I run `gradlew assemble` it works fine locally, it grabs the resources.

[the remark this answers is not in the document]
Comment (aanorbel) [reply composeApp/build.gradle.kts:277]: but you can launch the `dw` flavor

File: .github/workflows/validate.yml [hunk 6]
```diff
@@ -21,9 +22,9 @@ jobs:
         uses: ./.github/actions/setup
 
       - name: Build Android
-        run: ./gradlew assemble${{ matrix.type }}
+        run: ./gradlew copyBrandingToCommonResources assemble${{ matrix.type }} -Porganization=${{ matrix.organization }}
 
-      - name: Build iOS
+      - name: Build iOS Framework
```
Comment (sdsantos) [inline .github/workflows/validate.yml:27]: Should have the organization flag to no? Once we fix it...

**Pull request closed without merging**